rest-servlet/src/main/java/org/hawkular/inventory/rest/RestEnvironments.java

```diff
@@ -57,14 +60,16 @@
 
     @GET
     @Path("/{tenantId}/environments")
-    @ApiOperation("Returns all environments under given tenant.")
+    @ApiOperation("Returns all environments under given tenant. Accepts paging query parameters.")
```

Comment:
On 28 Apr 2015, at 10:59, Lukas Krejci wrote:

> Yeah, this is not ideal. I meant to find some "high level" place where 
> we would spec out the params once and only reference them like this in 
> the `@ApiOperation` docs. I obviously forgot to do that ;)

What if we put them in a page on hawkular.org and add a link here like

     @ApiOperation("Returns all environments under given tenant. Accepts 
paging query parameters. See 
http://hawkular.org/dev/rest-paging_params")